app/views/layouts/application.html.erb

```diff
@@ -11,6 +11,8 @@
   </head>
 
```

Comment:
- [ ] Please fix the following , I am trying to signup but I am I am not redirecting to the login and when I try to login with the created account I am getting an error saying invalid email please watch the below video 👇 
 

https://user-images.githubusercontent.com/75126481/217686534-5dd451bf-520d-41e4-84c6-9331f2c43795.mov